src/client/client_common.h

```diff
@@ -137,6 +137,30 @@ struct CloneSourceInfo {
     bool IsSegmentAllocated(uint64_t offset) const;
 };
 
+struct OpenContext {
+    int openflags;
+    std::string uuid;
+    OpenContext()
+        : openflags(CurveOpenFlags::CURVE_FORCE_WRITE) {}
+};
+
+static inline std::string TransformFlags(const int openflags) {
```

Comment:
remove `static`, and openflags can be a combinations of CurveOpenFlags, like `CURVE_EXCLUSIVE | CURVE_RDWR`, but, it seems this function doesn't handle this.